### PR TITLE
fix(overlay): update layout to improve UX on lower viewports

### PR DESCRIPTION
**Pull request by eliobricenov**
<--- Put the description here --->

### Summary of changes:
- made MM window non-floating
- added max-widths from https://www.figma.com/design/GKPELZxk551oVC3EDLgEDA/Current-Design?node-id=14888-158274&node-type=section&m=dev
- remove quest description unselect
- open the first quest description by default

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Comment by flavioislima** [comment]
@eliobricenov can you add screenshots, how to test and which screen resolutions have you tested?

**Review by flavioislima — APPROVED** [review]

**Comment by eliobricenov** [comment]
> @eliobricenov can you add screenshots, how to test and which screen resolutions have you tested?

Tested all screens down to steam-deck resolution (1300px)
image

File: src/frontend/components/UI/QuestsViewer/index.module.scss [hunk 1]
```diff
@@ -18,6 +12,14 @@
...
+.detailsWrapper {
+  width: 1045px;
+  min-width: 800px;
+  max-width: 1063px !important;
+
+  @media (max-width: 1280px) {
+    width: 540px;
+    min-width: 520px;
+    max-width: 1045px !important;
+  }
```
Comment (eliobricenov) [inline src/frontend/components/UI/QuestsViewer/index.module.scss:24]: For some reason, I couldn't use the break-points mixins, every time I wanted to import the `_variables.scss` I would get this error:

```
[plugin:vite:css] [sass] Undefined mixin.
```

even though we should be importing it with the `import '@hyperplay/ui/style.css'` we have in `index.tsx`

File: src/frontend/components/UI/QuestsViewer/index.module.scss [hunk 2]
```diff
@@ -18,6 +12,14 @@
...
+.detailsWrapper {
+  width: 1045px;
+  min-width: 800px;
+  max-width: 1063px !important;
+
+  @media (max-width: 1300px) {
+    width: 540px;
+    min-width: 520px;
+    max-width: 1045px !important;
+  }
```
Comment (eliobricenov) [inline src/frontend/components/UI/QuestsViewer/index.module.scss:24]: For some reason, I couldn't use the break-points mixins, every time I wanted to import the `_variables.scss` I would get this error:

```
[plugin:vite:css] [sass] Undefined mixin.
```

even though we should be importing it with the `import '@hyperplay/ui/style.css'` we have in `index.tsx`

**Review by jiyuu-jin — APPROVED** [review]
Code, LGTM

File: src/frontend/ExtensionManager/components/ExtensionContents/index.tsx [hunk 3]
```diff
@@ -22,36 +24,47 @@ const animation = {
...
+  return (
+    <webview
+      nodeintegrationinsubframes="true"
+      webpreferences="contextIsolation=true, nodeIntegration=true"
+      src={`chrome-extension://${extensionState.extensionId}/notification.html`}
+      className={ExtensionContentsStyles.mmWindow}
+      allowpopups={trueAsStr}
+    ></webview>
+  )
+}
```
Comment (BrettCleary) [inline src/frontend/ExtensionManager/components/ExtensionContents/index.tsx:49]: could maybe combine these and pass in the `notification.html` or `popup.html` to keep it dry

**Review by BrettCleary — APPROVED** [review]
looks great

File: package.json
Comment (BrettCleary) [inline package.json:1]: if you want to hotfix, go ahead and bump the version to 0.19.2

File: src/frontend/ExtensionManager/components/ExtensionContents/index.tsx (hunk 3 above)
Comment (eliobricenov) [reply src/frontend/ExtensionManager/components/ExtensionContents/index.tsx:49]: I think the code duplication is fine in this case

**Review by nyghtstalker — APPROVED** [review]
Confirmed that when the screen is reduced MM does not block the claim button.

image

**Pull request merged**